signer/apk/apk_test.go

```diff
@@ -5,16 +5,44 @@ import (
 	"bytes"
 	"encoding/base64"
 	"encoding/pem"
+	"fmt"
 	"io/ioutil"
 	"os"
 	"os/exec"
 	"strings"
 	"testing"
+	"github.com/pkg/errors"
 
 	"go.mozilla.org/autograph/signer"
 )
 
+func TestValidateZIPOption(t *testing.T) {
+	var testcases = []struct {
+		input    string
+		result   error
+	}{
+		{"all", nil},
+		{"passthrough", nil},
+		{"", errors.New("Invalid APK ZIP option")},
+	}
+	for i, testcase := range testcases {
+		testcase := testcase
+		t.Run(fmt.Sprintf("GetOptions (%d) input: %#v", i, testcase.input), func (t *testing.T) {
```

Comment:
I don't follow the need for `t.Run` here. You're already inside of a unit test, couldn't you just go remove this and go straight to line 33?

Reply:
This is from: https://golang.org/pkg/testing/#hdr-Subtests_and_Sub_benchmarks

```golang
func TestGroupedParallel(t *testing.T) {
    for _, tc := range tests {
        tc := tc // capture range variable
        t.Run(tc.Name, func(t *testing.T) {
            t.Parallel()
            ...
        })
    }
}
```

Reply:
Nice! I wasn't aware of this pattern, but definitely :+1:

Reply:
Added a comment since it is weird.